networking/networking_operators/metallb-operator/metallb-upgrading-operator.adoc

```diff
@@ -6,21 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-If you are currently running version 4.10 or an earlier version of the MetalLB Operator, please note that automatic updates to any version later than 4.10 do not work. Upgrading to a newer version from any version of the MetalLB Operator that is 4.11 or later is successful. For example, upgrading from version 4.12 to version 4.13 will occur smoothly.
+You can upgrade from an earlier version of the MetalLB Operator to a newer version. 
 
-A summary of the upgrade procedure for the MetalLB Operator from 4.10 and earlier is as follows:
-
-. Delete the installed MetalLB Operator version for example 4.10. Ensure that the namespace and the `metallb` custom resource are not removed.
-
-. Using the CLI, install the MetalLB Operator {product-version} in the same namespace where the previous version of the MetalLB Operator was installed.
+As a cluster administrator, start the upgrade process by deleting the MetalLB Operator by using the OpenShift CLI (`oc`) or the web console.
 
 [NOTE]
 ====
-This procedure does not apply to automatic z-stream updates of the MetalLB Operator, which follow the standard straightforward method.
+The upgrading procedure does not apply to automatic z-stream updates of the MetalLB Operator.
```

Comment:
This may not be relevant if it was due to removal of previous version of operator.

The upgrade of operator of x stream releases is same as z stream releases.

Reply:
Thanks. Removing this statement would be good as it was getting complex with the "standard straightforward" reference.